CMSgov/mint-app
# Easi 2937/active and ended to model status

# EASI-2937

## Changes and Description

- Added Active and Ended Model Status on the backend
- Update Status translations on the frontend

## How to test this change
1. Bring up the application
2. Make a model
3. Change the status to active
4. Change the status to ended
5. ensure data persists and that spelling is correct.

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
